oxd-server/src/main/java/org/gluu/oxd/server/service/ValidationService.java

```diff
@@ -41,10 +41,14 @@ public void notBlankOpHost(String opHost) {
     public Pair<Rp, Boolean> validate(IParams params) {
         notNull(params);
         if (params instanceof HasOxdIdParams) {
-            validate((HasOxdIdParams) params);
+            if (!isInstanceOfGetRpParamsWithList(params)) {
```

Comment:
it look good but would you be so kind to increase readibility ? 
Put 
```
notNull(params);
if (isInstanceOfGetRpParamsWithList(params)) {
    return new Pair(null, true);
}
// rest of the code here
```
Then it's one `if` without inner ifs and in one place only.

Reply:
Changes done